rules/0265-php_rules.xml

```diff
@@ -59,9 +59,6 @@
     <match> expects parameter 1 to be string, array given in</match>
     <group>attack,pci_dss_6.5,pci_dss_11.4,gdpr_IV_35.7.d,nist_800_53_SA.11,nist_800_53_SI.4,tsc_CC6.6,tsc_CC7.1,tsc_CC8.1,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
     <description>PHP web attack.</description>
```

Comment:
Probably this one also pairs with  tags T1190 "Exploit Public-Facing Application" and T107.001 "Application Layer Protocol: Web Protocols"